composer.json

```diff
@@ -75,7 +75,7 @@
     },
     "require-dev": {
         "aws/aws-sdk-php": "~3.0",
-        "iron-io/iron_mq": "~2.0",
+        "iron-io/iron_mq": "~2.0 || ~4.0",
```

Comment:
We always use the `|` syntax. See our master branch. We'd like to make sure stuff works on older versions of composer.